Sources/Kitura/bodyParser/BodyParser.swift

```diff
@@ -21,13 +21,31 @@ import Foundation
 
 // MARK: BodyParser
 
-/// The `BodyParser` parses the body of the request prior to sending it to the handler. It reads the Content-Type of the message header and populates the `RouterRequest` body field with a `ParsedBody` enumeration (e.g. json, raw, text, urlEncoded).
+/// The `BodyParser` parses the body of the request prior to sending it to the handler. It reads the Content-Type of the message header and populates the `RouterRequest` body field with a corresponding `ParsedBody` enumeration.
 /// In order for the BodyParser to be used it must first be registered with any routes that are interested in the ParsedBody payload.
-///### Usage Example: ###
-/// In this example, all routes to the BodyParser middleware are registered to the `BodyParser` middleware.
-///```swift
-///   router.all("/*", middleware: BodyParser())
-///```
+/// ### ParsedBody enumerations: ###
+/// ```swift
+/// "application/json" -> JSON: [String: Any]
+/// "text/*" -> text: String
+/// "application/x-www-form-urlencoded" -> URLEncoded: [String:String]
+/// "multipart/*" -> multiPart: [`Part`]
```

Comment:
I find this formatting confusing - I think what we're trying to convey here is the mapping from the incoming `Content-Type` to our internal representation of the body data.  How about an opening statement such as:
> This represents the mapping from the incoming `Content-Type` to an internal representation of the body data, as follows:
> - "application/json": `.json([String: Any])`
> - "text/*": `.text(String)`
> - ...etc
>
> Any other content type maps to the `.raw(Data)` case.
>
> For each case, there is a corresponding convenience property for accessing the associated data - for example, `asJSON` returns a `[String: Any]?`.